helm/cas-ciip-portal/Chart.yaml

```diff
@@ -8,3 +8,7 @@ dependencies:
 - name: cas-postgres
   version: "0.4.0"
   repository: https://bcgov.github.io/cas-postgres/
+- name: mailhog
+  version: "3.2.0"
+  repository: "file://../mailhog"
```

Comment:
If the chart works as-is, or just requires some change to the docker image to support anyuid, then I would use it from that repository instead of copying it in this repo